misc/aspell_dict

```diff
@@ -475,6 +481,7 @@ refundTokens
 relayer
 relayerModule
 relayers
+releaseCapability
```

Comment:
Is this a new API that wasn't originally covered in the ADR?

Reply:
Yes, alas I had forgotten about this particular function when writing the ADR.

https://github.com/cosmos/cosmos-sdk/pull/5873